glue_jupyter/common/state_widgets/viewer_histogram.vue

```diff
@@ -4,28 +4,39 @@
             <v-select :items="x_att_items" label="x axis" v-model="x_att_selected"/>
         </div>
         <div>
-            <v-btn-toggle dense multiple :value="modeSet" @change="modeSetChange">
-
-                <v-tooltip bottom>
+            <v-text-field type="number" step="1" label="number of bins" v-model="glue_state.hist_n_bin" />
+        </div>
+        <div>
+            <glue-float-field label="x-min" :value.sync="glue_state.hist_x_min" />-->
+        </div>
+        <div>
+            <glue-float-field label="x-max" :value.sync="glue_state.hist_x_max" />-->
+        </div>
+        <div>
+            <v-toolbar title="Toolbar">
+                <v-toolbar-items>
+                  <v-tooltip>
                     <template v-slot:activator="{ on }">
-                        <v-btn v-on="on" small value="normalize">
-                            <v-icon>unfold_more</v-icon>
-                        </v-btn>
-                    </template>
+                         <v-btn v-on="on" small value="normalize">
+                             <v-icon>unfold_more</v-icon>
+                         </v-btn>
+                     </template>
                     <span>normalize</span>
-                </v-tooltip>
-
-                <v-tooltip bottom>
+                  </v-tooltip>
+                  <v-tooltip bottom>
                     <template v-slot:activator="{ on }">
                         <v-btn v-on="on" small value="cumulative">
                             <v-icon>trending_up</v-icon>
                         </v-btn>
                     </template>
                     <span>cumulative</span>
-                </v-tooltip>
-            </v-btn-toggle>
-        </div>
+                  </v-tooltip>
 
+                  <v-btn variant="outlined" small @click="axis_to_bins">
+                      Fit Axes to Bins
```

Comment:
I think this should be ``Update bins to view`` - it's the bins that are being updated not the axes. And this would then match the glue-qt wording.